lib/cldr/export/data/rbnf.rb

```diff
@@ -11,17 +11,16 @@ def initialize(*args)
         end
 
         def rule_groups
-          if File.exist?(path)
-            select("rbnf/rulesetGrouping").map do |grouping_node|
-              {
-                :type => grouping_node.attribute("type").value,
-                :ruleset => (grouping_node / "ruleset").map do |ruleset_node|
-                  rule_set(ruleset_node)
-                end
-              }
-            end
-          else
-            {}
+          grouping_nodes = select("rbnf/rulesetGrouping")
+          return {} if grouping_nodes.empty?
```

Comment:
I made this change since it didn't make sense to check for the existence of `path` anymore.

This had a side-effect of cleaning up a edge-case bug:

`common/rbnf/en_001.xml` exists, but is empty. So this code was returning `[]` instead of the expected `{}`. Now that it has been fixed to return `{}`, `en-001/rbnf.yml` is no longer output since [we don't output empty hashes](https://github.com/ruby-i18n/ruby-cldr/issues/85).